packages/mobile/src/invite/saga.ts

```diff
@@ -81,18 +81,26 @@ export function getInvitationVerificationFeeInWei() {
 }
 
 export async function generateLink(inviteCode: string, recipientName: string) {
-  const packageName = VersionCheck.getPackageName().replace(/\.debug$/g, '.integration')
-  const playStoreLink = await VersionCheck.getPlayStoreUrl({ packageName })
+  const androidPackageName = VersionCheck.getPackageName().replace(/\.debug$/g, '.integration')
+  const playStoreLink = await VersionCheck.getPlayStoreUrl({ androidPackageName })
   const referrerData = encodeURIComponent(`invite-code=${inviteCode}`)
-  const referrerLink = `${playStoreLink}&referrer=${referrerData}`
-  const shortUrl = await dynamicLink(referrerLink)
-  const msg = i18n.t('sendFlow7:inviteSMS', {
+  const playStoreLinkReferrerLink = `${playStoreLink}&referrer=${referrerData}`
+
+  const iOSBundleId = '1482389446' // This is the id in the Apple Store URL
```

Comment:
Couldn't figure out a way to get it programatically, so up to suggestions. Moving to an `.env` file or at least `conf.ts` feels a happier place if there's no function that yields this number.